src/basilisp/lang/runtime.py

```diff
@@ -1113,7 +1113,10 @@ def apply_kw(f, args):
 
 def count(coll) -> int:
     try:
-        return len(coll)
+        if coll:
+            return len(coll)
+        else:
+            return 0
```

Comment:
```suggestion
    if coll is None:
        return 0
    try:
        return len(coll)
```

Most Python objects will respond to the implicit `__bool__` which is called via `if obj:` checks, but I do not imagine we want to try to return 0 for those cases. I prefer an explicit `is None` check, which should be fairly cheap since it's an identity check. Also note, I suggest moving that out of the `try`/`except` block.

Reply:
Correct, updated.